### ISSUE TITLE
parser.times(None) does not work

### ISSUE THREAD

**Issue by melak47** [python-parsy/parsy#21]
The definition of `times` reads:

```py
def times(self, min, max=None):
    # max=None means exactly min
    # min=max=None means from 0 to infinity
```

But setting min=max=None doesn't actually work. `many` calls ` self.times(0, float('inf'))` to get the zero-to-infinity behavior -- the comment is probably out of date and should be removed if that's the case.

**Comment by spookylukey** [comment]
Yes, looks like you are right. The 0 to infinity behaviour is more easily accessible as `many`.

**Comment by spookylukey** [comment]
Fixed in 17cbdb0a5bf267afe008c85bc463b081072f95a0 - thanks for letting us know.
